src/commands/java_maven_build_test.yml

```diff
@@ -0,0 +1,31 @@
+description: >
+  Execute a maven build including integration tests for a java app.
+  Supports app-module from monorepo as well as single app from single-app-repo.
+
+parameters:
+  app_path:
+    description: "Path of maven module for the app, or \".\" for single-app-repo"
+    type: string
+    default: "."
```

Comment:
for monorepo you need to overwrite this param (we cant know the correct path otherwise). the default here is for the non-monorepo case (and allows to ignore the param in those configurations)